mkt/site/tests/test_monitor.py

```diff
@@ -1,20 +1,68 @@
 import time
 
 from django.conf import settings
+from django.test.utils import override_settings
 
 from mock import Mock, patch
-from nose.tools import eq_
+from nose.tools import eq_, ok_
+import redisutils
 import requests
 
 import mkt.site.tests
-from mkt.site.monitors import receipt_signer as signer, package_signer
+from mkt.site import monitors
 
 
 @patch.object(settings, 'SIGNED_APPS_SERVER_ACTIVE', True)
 @patch.object(settings, 'SIGNING_SERVER', 'http://foo/')
 @patch.object(settings, 'SIGNED_APPS_SERVER', 'http://baz/')
 class TestMonitor(mkt.site.tests.TestCase):
-    # Some rudimentary tests for the rest of the monitor would be nice.
+    # Would be nice to tests what happens when test_memcache(),
+    # test_libraries(), test_elastic(), test_path() and test_redis() fail.
+    @patch('socket.socket')
+    def test_memcache(self, mock_socket):
+        mocked_caches = {
+            'default': {
+                'BACKEND': 'caching.backends.memcached.MemcachedCache',
+                'LOCATION': '127.0.0.1:6666'
+            }
+        }
+        cache_info = mocked_caches['default']['LOCATION'].split(':')
+        mock_socket_instance = Mock()
+        mock_socket.return_value = mock_socket_instance
+        with override_settings(CACHES=mocked_caches):
+            status, memcache_results = monitors.memcache()
+            eq_(status, '')
+            eq_(mock_socket_instance.connect.call_args[0][0][0], cache_info[0])
```

Comment:
It's hard to understand at a glance what this comparison is. Maybe you could use the msg arg to eq_ for some more info?